<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>comForm</class>
 <widget class="QWidget" name="comForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="openClose_Button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>135</width>
     <height>128</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="com_label">
        <property name="text">
         <string>串口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="baud_label">
        <property name="text">
         <string>波特率</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="parity_label">
        <property name="text">
         <string>校验位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="data_label">
        <property name="text">
         <string>数据位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="stop_label">
        <property name="text">
         <string>停止位</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="com_Box"/>
      </item>
      <item>
       <widget class="QComboBox" name="baud_Box">
        <item>
         <property name="text">
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="parity_Box">
        <item>
         <property name="text">
          <string>NONE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ODD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EVEN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MARK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SPACE</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="data_Box">
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="stop_Box">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>stop_label</zorder>
  <zorder>baud_label</zorder>
  <zorder>com_label</zorder>
  <zorder>data_label</zorder>
  <zorder>parity_label</zorder>
  <zorder>stop_Box</zorder>
  <zorder>data_Box</zorder>
  <zorder>baud_Box</zorder>
  <zorder>com_Box</zorder>
  <zorder>parity_Box</zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder>openClose_Button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
